test/sbom/cyclonedx_test.exs

```diff
@@ -85,5 +85,96 @@ defmodule SBoM.CycloneDXTest do
       assert xml =~
                ~s(<hashes><hash alg="SHA-256">fdf843bca858203ae1de16da2ee206f53416bbda5dc8c9e78f43243de4bc3afe</hash></hashes>)
     end
+
+    test "generates CycloneDX in json format if specified" do
+      json =
+        [
+          %{
+            type: "library",
+            name: "name",
+            version: "0.0.1",
+            description: "support for CycloneDX in json support",
+            purl: "pkg:hex/name@0.0.1",
+            licenses: ["Apache-2.0"]
+          }
+        ]
+        |> bom(format: :json)
+        |> to_string()
+
+      assert json =~ "{\n  \"bomFormat\": \"CycloneDX\""
```

Comment:
It seems a bit brittle to assert on a JSON literal with whitespace. I would prefer asserting that the result can be decoded by Jason, and then assert on values in the resulting map:

```elixir
  assert {:ok, decoded} = Jason.parse(json)
  assert decoded["bomFormat"] == "CycloneDX"
```

Same thing for the other JSON assertions in this file.